ProjectLighthouse/Types/Users/PrivacyType.cs

```diff
@@ -21,6 +21,17 @@ public enum PrivacyType
 
 public static class PrivacyTypeExtensions
 {
+    public static string ToReadableString(this PrivacyType type, string area)
```

Comment:
Should this be translated

Reply:
I have no problem adding translations for it. One moment